_topic_map.yml

```diff
@@ -722,19 +722,19 @@ Topics:
   File: setting-up-trusted-ca
   Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
 ---
-#Name: OpenShift Pipelines
-#Dir: openshift_pipelines
-#Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#Topics:
+Name: OpenShift Pipelines
+Dir: openshift_pipelines
+Distros: openshift-enterprise,openshift-webscale,openshift-origin
+Topics:
 #- Name: Understanding OpenShift Pipelines
 #  File: understanding-openshift-pipelines
 #- Name: Installing Pipelines
 #  File: installing-pipelines
 #- Name: Getting started with OpenShift Pipelines
 #  File: getting-started-with-openshift-pipelines
-#- Name: Working with Pipelines using the Developer perspective
-#  File: working-with-pipelines-using-developer-perspective
-#---
+- Name: Working with Pipelines using the Developer perspective
+  File: working-with-pipelines-using-the-developer-perspective
```

Comment:
We can drop the articles in file names. But if you decide to change it, please ensure that all the ID and context values are changed too in the working-with-pipelines-using-the-developer-perspective.adoc

Reply:
changed the IDs